db/languages.yml

```diff
@@ -52,7 +63,7 @@ java:
     java6:
       :name: Java 1.6
       :compiler: /usr/bin/javac;/usr/bin/jar
-      :compiler_command: '%{compiler[0]} -O -source 1.6 -J-Xms16m -J-Xmx256m %{source} && %{compiler[1]} cfe %{output} Main -C %{source_dir} Main.class'
+      :compiler_command: '%{compiler[0]} -version 1>&2 && %{compiler[1]} --version 1>&2 && %{compiler[0]} -O -source 1.6 -J-Xms16m -J-Xmx256m %{source} && %{compiler[1]} cfe %{output} Main -C %{source_dir} Main.class'
```

Comment:
I don't think anybody cares about the version of `jar`.

Reply:
Fair enough